pkg/datarepair/checker/checker.go

```diff
@@ -28,7 +27,7 @@ type Config struct {
 
 // Run runs the checker with configured values
 func (c Config) Run(ctx context.Context, server *provider.Provider) (err error) {
-	defer mon.Task()(&ctx)(&err)
+	defer datarepair.Mon.Task()(&ctx)(&err)
```

Comment:
This looks weird. Why did you move the `var mon = monkit.Package()` declaration to datarepair/common.go? Why not under the same package, i.e. checker/common.go?

Reply:
I can move it to checker/common. Thought to move it to the data repair checker since all of the packages within that use mon.task, but I do agree it is clunky